frameworks/helloworld/src/test/java/com/mesosphere/sdk/helloworld/scheduler/ServiceTest.java

```diff
@@ -329,23 +329,16 @@ public String getDescription() {
 
     @Test
     public void testWorldDecommissionDefaultExecutor() throws Exception {
-        testWorldDecommission(true);
-    }
-
-    @Test
-    public void testWorldDecommissionCustomExecutor() throws Exception {
-        testWorldDecommission(false);
+        testWorldDecommission();
     }
 
     /**
      * Tests scheduler behavior when the number of {@code world} pods is reduced.
      */
-    private void testWorldDecommission(boolean useDefaultExecutor) throws Exception {
+    private void testWorldDecommission() throws Exception {
```

Comment:
I'd just condense this into the test method itself. It was only separate to handle switching which was used.